config/environments/production.rb

```diff
@@ -100,30 +82,4 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
-
-  config.action_mailer.delivery_method = :notify
-  config.action_mailer.notify_settings = {
-    api_key: Rails.application.secrets.notify_api_key,
-  }
```

Comment:
I think you'll need to keep these lines, as they tell the application to use GOV.UK Notify to send emails, instead of the default mailer.